packages/toast/src/ToastContext/ToastContext.types.ts

```diff
@@ -20,6 +20,8 @@ export interface ToastContextProps {
   getToast: (id: ToastId) => ToastProps | undefined;
   getStack: () => ToastStack | undefined;
   clearStack: () => void;
+  /** @private */
+  _hasProvider?: boolean;
```

Comment:
Do we need this internal flag? Could we detect it by calling `useToastContext` and `console.warn`ing if the right variables aren't defined?

Reply:
All the vars will be defined, but I changed it to check whether `getStack` returns something